test/io/ProcessTest.ooc

```
@@ -39,6 +41,11 @@ ProcessTest: class extends Fixture {
 			reader close() . free()
 		})
 	}
+	_createOutputDirectory: func {
+		file := File new("test/io/output")
+		file mkdir()
+		file free()
+	}
```

Nothing, it silently ignores.